support/Support.cpp

```diff
@@ -0,0 +1,70 @@
+/*
+  Copyright (c) 2021-present, Trail of Bits, Inc.
+  All rights reserved.
+
+  This source code is licensed in accordance with the terms specified in
+  the LICENSE file found in the root directory of this source tree.
+*/
+
+#include "SpecFilePaths.h"
+
+#include "Support.h"
+
+namespace sleigh {
+
+namespace {
+
+std::optional<std::filesystem::path>
+FindSpecFileInSearchPath(std::string_view file_name,
+                         std::filesystem::path search_path) {
+  search_path.append("Ghidra").append("Processors");
+  // Check whether a SLEIGH installation exists at this path
+  if (!std::filesystem::is_directory(search_path)) {
+    return {};
+  }
+  // Each directory under Processors/ represents a family of architectures
+  //
+  // Spec files should reside under:
+  // <install_prefix>/Processors/<arch>/data/languages
```

Comment:
I think this should read more like 
```suggestion
  // <root_sla_dir>/Ghidra/Processors/<arch>/data/languages
```

Reply:
Not sure if you want to also change the line
```diff
-  // Each directory under Processors/ represents a family of architectures
+  // Each directory under Ghidra/Processors/ represents a family of architectures
```